pkg/ovsdb/client/client.go

```diff
@@ -80,45 +82,76 @@ func NewNbClient(addr string) (client.Client, error) {
 			RootCAs:            certPool,
 			InsecureSkipVerify: true,
 		}
-
 		options = append(options, client.WithTLSConfig(tlsConfig))
 	}
 
-	c, err := client.NewOVSDBClient(dbModel, options...)
-	if err != nil {
+	switch db {
+	case NBDB:
```

Comment:
Avoid using `switch...case...`. Anything we need can be passed in as parameters.

Reply:
ok